cmd/compute-domain-controller/daemonset.go

```diff
@@ -167,7 +168,7 @@ func (m *DaemonSetManager) Stop() error {
 	return nil
 }
 
-func (m *DaemonSetManager) Create(ctx context.Context, namespace string, cd *nvapi.ComputeDomain) (*appsv1.DaemonSet, error) {
+func (m *DaemonSetManager) Create(ctx context.Context, cd *nvapi.ComputeDomain) (*appsv1.DaemonSet, error) {
 	ds, err := getByComputeDomainUID[*appsv1.DaemonSet](ctx, m.mutationCache, string(cd.UID))
```

Comment:
Question -- could we not replace this code block with a call to `m.Get(ctx, string(cd.UID))`?